app/src/main/java/com/github/epfl/meili/MainActivity.kt

```diff
@@ -4,16 +4,26 @@ import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.View
+
 import com.github.epfl.meili.home.GoogleSignInActivity
+import com.github.epfl.meili.forum.ForumActivity
+
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
     }
 
-    fun onClickLaunchSignInView(view: View){
+
+    fun onClickLaunchSignInView(view: View) {
         val intent = Intent(this, GoogleSignInActivity::class.java)
         startActivity(intent)
     }
+
+    /** Called when the user taps the forum button */
+    fun openForum(view: View) {
```

Comment:
From my opinion it would be great to have 1 test for this button even if it's just temporary. On the MainActivityTest class there is already a Test made for the launchSignInView, you could reuse it.